pipeline/scripts/inference_pipeline.jl

```diff
@@ -0,0 +1,41 @@
+using DrWatson
+@quickactivate "Analysis pipeline"
+
+# include AnlysisPipeline module
+include(srcdir("AnalysisPipeline.jl"))
+
+@info("""
+      Running inference on truth data.
+
+      ---------------------------------------------
+      Currently active project is: $(projectname())
+      Path of active project: $(projectdir())
+      """)
+
+## Set up the truth Rt and save a plot of it
+using .AnalysisPipeline, Plots, JLD2, EpiAware, Distributions
+
+wkly_ar = AR(
+    damp_priors = [truncated(Normal(0.8, 0.05), 0, 1)],
+    std_prior = HalfNormal(1.0),
+    init_priors = [Normal(0.0, 0.25)]
+) |> ar -> BroadcastLatentModel(ar, 7, RepeatBlock())
```

Comment:
can we be flash and not repeat the broadcast (so have a tuple of models and then broadcast them all using map)

Reply:
I'm all in favour of flash!

Reply:
Think I've managed to do this in latest commit